fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -1577,6 +1740,266 @@ private static void generateBatchPathItem(JsonObjectBuilder path) {
+        response.add("description", "Export job completed");
+
+        // add response body
+        JsonObjectBuilder content = factory.createObjectBuilder();
+        JsonObjectBuilder contentType = factory.createObjectBuilder();
+        schema = factory.createObjectBuilder();
+        schema.add("type", "object");
+        JsonObjectBuilder properties = factory.createObjectBuilder();
+        JsonObjectBuilder transactionTime = factory.createObjectBuilder();
+        transactionTime.add("type", "string");
+        properties.add("transactionTime", transactionTime);
+        
+        JsonObjectBuilder request = factory.createObjectBuilder();
+        request.add("type", "string");
+        properties.add("request", request);
+
+        JsonObjectBuilder requiresAccessToken = factory.createObjectBuilder();
+        requiresAccessToken.add("type", "boolean");
+        properties.add("requiresAccessToken", requiresAccessToken);
+        
+        JsonObjectBuilder output = factory.createObjectBuilder();
+        output.add("type", "array");
+        JsonObjectBuilder items = factory.createObjectBuilder();
+        items.add("type", "object");
+        JsonObjectBuilder itemProperties = factory.createObjectBuilder();
+        
+        JsonObjectBuilder outputArrayEntryTypeProperty = factory.createObjectBuilder();
+        outputArrayEntryTypeProperty.add("type", "string");
+        itemProperties.add("type", outputArrayEntryTypeProperty);
+
+        JsonObjectBuilder outputArrayEntryUrlProperty = factory.createObjectBuilder();
+        outputArrayEntryUrlProperty.add("type", "string");
+        itemProperties.add("url", outputArrayEntryUrlProperty);
+        
+        JsonObjectBuilder outputArrayEntryCountProperty = factory.createObjectBuilder();
+        outputArrayEntryCountProperty.add("type", "integer");
+        itemProperties.add("count", outputArrayEntryCountProperty);
+        
+        items.add("properties", itemProperties);
+        output.add("items", items);
+        properties.add("output", output);
+        
+        schema.add("properties", properties);
+        contentType.add("schema", schema);
+        content.add("application/json", contentType);
+
+        response.add("content", content);
+        responses.add("200", response);
+        httpMethodBuilder.add("responses", responses);
+
+        path.add("get", httpMethodBuilder);
+    }
+    
+    /**
+     * Generates the $everything path for a Patient resource
+     *  
+     * @param path
+     * @param addIdParam
+     */
+    private static void generatePatientEverythingPathItem(JsonObjectBuilder path, boolean addIdParam) {
+        JsonObjectBuilder httpMethodBuilder = factory.createObjectBuilder();
+
+        JsonArrayBuilder tags = factory.createArrayBuilder();
+        tags.add("Patient");
+
+        httpMethodBuilder.add("tags", tags);
+        httpMethodBuilder.add("summary", "Return all the information related to one or more patients");       
```

Comment:
this summary is good for `[base]/Patient/$everything` but 
it would be nice if the `[base]/Patient/[id]/$everything` variant said "Return all the information related to a given patient" instead